lib/pdk/cli/build.rb

```diff
@@ -28,8 +28,12 @@ module PDK::CLI
       #
       unless module_metadata.forge_ready?
         if opts[:force]
-          PDK.logger.error _('This module is missing required fields in the metadata.json. Re-run the build command without --force to add this information.')
-          exit 1
+          PDK.logger.warn _(
+            'This module is missing the following fields in the metadata.json: %{fields}. ' \
+            'These missing fields may affect the visibility of the module on the Forge.',
```

Comment:
Are any of the fields things that would cause the Forge to reject it? If so maybe this should say "may affect your ability to publish this module on the Forge" instead?